about.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+title: About
+#external-link: //pelux.io/software-factory
+permalink: /about/
+---
+
+## Modular Software Platform
+
+PELUX is based around an extensive set of largely independent software components for automotive applications. Through the addition of key platform components enabling dynamic service access control and sandboxing technologies, it is possible to integrate services into the platform and choose which interfaces to expose for specific applications. This makes it possible to take full benefit of open source software and produce dynamic content without having to modify implementations or create maintenance requirements.
+
+## Change the Way You Work
+
+### Software Factory
+Luxoft provides a Software Factory to help you ease through development pains. The Software Factory (SWF) concept is intended to be used as a structured way to set up a platform, and is considered a handbook for the work needed to develop that platform. This concept also invites users to start creating and populating their own SWFs as new projects arise.
+ 
+The SWF documentation contains processes (both technical and for project management) and instructions on how to work with PELUX, as well as instructions on how to create a standalone project based on PELUX. This ensures knowledge is transparently shared across teams for better collaboration and reduced development time.
+
+### Dynamic Content
+Modern infotainment systems are built on platform firmware with selected third-party software by leading vendors, augmented with apps delivered through app stores. PELUX securely decouples firmware from apps and third-party services deployment through its Containment and Platform Access Manager components. These technologies limit what each dynamically added piece of software can access, but also what capabilities it can provide to the rest of the system. The Application Manager component is responsible for life cycle management, including interacting with leading app stores and managing security. It also ensures apps work with the rest of the platform and are integrated seamlessly, creating a rich and fresh user experience.
```

Comment:
The way AppMan is mentioned makes it sound to me like it is part of PELUX. Do we want to imply that?

Reply:
The text comes from the PELUX factsheet and to be consistent I'm not keen to change this at this point. I agree that it could be more clear that it's the **Qt** Application Manager and it's only relevant when building PELUX with the Neptune 3 reference UI.